scripts/rebuild-testbench.sh

```diff
@@ -56,6 +56,8 @@ setup_xtensa_tools_build()
     BUILD_TYPE=xt
     BUILD_TARGET=
     BUILD_DIR_NAME=build_xt_testbench
+    COMPILER="xt-xcc"
+    LDEXTRAFLAGS="-Wl,-LE"
```

Comment:
This flag never had any effect because there is no `./E/` directory. I'm dropping it in #8384

Reply:
(facepalm) good that you spotted it, I was yesterday trying to find from manual what it's doing :)